src/main/java/frc/robot/commands/Arm/SpinAndSwing.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.Arm;
+import frc.robot.subsystems.ArmSubsystem;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.commands.Arm.SpinNoteContainerMotor;
+import frc.robot.commands.Arm.StopNoteContainerMotor;
+import frc.robot.commands.Arm.SwingForwardServo;
+import frc.robot.commands.Arm.SwingBackServo;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class SpinAndSwing extends SequentialCommandGroup {
+
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+
+  public SpinAndSwing(ArmSubsystem armSubsystem) {
+    addCommands(
+        new SpinNoteContainerMotor (armSubsystem, 100, 150),
+        Commands.waitSeconds(.5), 
```

Comment:
Maybe SpinNoteContainerMotor can run until it detects it gets up to speed, by checking if the RPMs are stabilized or above a min value.  Then we don't need to guess how many seconds - it just goes when ready